main.py

```diff
@@ -30,46 +30,93 @@ def hello():
-def get_circle(circle_id):
+    if not resource:
+        # Request specific user.
+        return jsonify(person.json_repr())
+
+    # Request specific resource associated with the person
+    output = []
+    if resource not in ['circles', 'events', 'knows']:
+        abort(404, description="Invalid resource specified")
+
+    if resource == 'circles':
+        for c in person.IsMember:
+            output.append(c.json_repr())
+    if resource == 'events':
+        for e in person.InvitedTo:
+            output.append(e.json_repr())
+    if resource == 'knows':
+        for k in person.Knows:
+            output.append(k.json_repr())
+    return jsonify(output)
+
+
+@app.route('/circles/api/v1.0/circles/<int:circle_id>/', defaults={'resource': None})
+@app.route('/circles/api/v1.0/circles/<int:circle_id>/<resource>', methods=['GET'])
+def get_circle(circle_id, resource):
     # Request all circles.
     if not circle_id:
         return jsonify(Circle=get_all_nodes(Circle))
 
-    # Request specific circle.
+    # Fetch circle.
     circle = Circle.match(graph, circle_id).first()
     if not circle:
         abort(404, description="Resource not found")
-    return jsonify(Circle=circle.json_repr())
-
-
-@app.route('/circles/api/v1.0/events/', defaults={'event_id': None})
-@app.route('/circles/api/v1.0/events/<int:event_id>', methods=['GET'])
-def get_event(event_id):
+    if not resource:
+        # Request specific circle
+        return jsonify(circle.json_repr())
+
+    # Request specific resource associated with the circle
+    output = []
+    if resource not in ['members', 'events']:
+        abort(404, description="Invalid resource specified")
+
+    if resource == 'members':
+        for m in circle.HasMember:
+            output.append(m.json_repr())
+    if resource == 'events':
+        for e in circle.Scheduled:
+            output.append(e.json_repr())
+    return jsonify(output)
+
+
+@app.route('/circles/api/v1.0/events/<int:event_id>/', defaults={'resource': None})
+@app.route('/circles/api/v1.0/events/<int:event_id>/<resource>', methods=['GET'])
+def get_event(event_id, resource):
     # Request all events.
     if not event_id:
         return jsonify(Event=get_all_nodes(Event))
 
-    # Request specific event.
+    # Fetch event.
     event = Event.match(graph, event_id).first()
     if not event:
         abort(404, description="Resource not found")
-    return jsonify(Event=event.json_repr())
+    if not resource:
+        # Request specific event.
+        return jsonify(event.json_repr())
+
+        # Request specific resource associated with the circle
+    output = []
+    if resource not in ['invitees', 'circle']:
+        abort(404, description="Invalid resource specified")
+
+    if resource == 'circle':
+        for c in event.BelongsTo:
+            output.append(c.json_repr())
+    if resource == 'invitees':
+        for p in event.Invited:
+            output.append(p.json_repr())
```

Comment:
Can you check the API on this? I think we want to return whether the person is attending or not as well, right? So maybe just a list of [{person_id : True}, {person_id: False}]. Idk about this exact format.